service/internal/graph/graph.go

```diff
@@ -196,6 +196,23 @@ func (g *Graph) createNodes(set Settings) error {
 	return nil
 }
 
+// formatPipelineNameWithSignal formats pipeline signal types with their names.
+func formatPipelineNameWithSignal(pipelineIDs []pipeline.ID, signal pipeline.Signal) string {
+	var formatted string
+
+	for _, pid := range pipelineIDs {
+		if pid.Signal().String() == signal.String() {
+			if name := pid.Name(); name == "" {
+				formatted = pid.Signal().String()
+			} else {
+				formatted = fmt.Sprintf("%s/%s", pid.Signal().String(), name)
+			}
```

Comment:
Is this logic not the same as https://github.com/open-telemetry/opentelemetry-collector/blob/main/pipeline/pipeline.go#L106?